report/v0.2.0/openrouter_google_palm-2-chat-bison-32k@golang@@golang-plain

```diff
@@ -0,0 +1,58 @@
+2024/04/03 22:54:19 Evaluating model "openrouter/google/palm-2-chat-bison-32k" using language "golang" and repository "/home/zimmski/symflower/src/github.com/symflower/eval-symflower-codegen-testing/testdata/golang/plain"
```

Comment:
Now that i see this. Can we put the files in directories instead: So `report/v0.2.0/openrouter_google_palm-2-chat-bison-32k@golang@@golang-plain` becomes

`$path-for-reports/$version/$model/$language/$repository.log`

Sorry for the change.